CitiesUIService/src/main/java/com/example/util/SwaggerConfig.java

```diff
@@ -1,5 +1,6 @@
 package com.example.util;
 
+import com.google.common.base.Predicates;
```

Comment:
please don't create util folders, it'll be a heap of rubbish eventually. Try to understand what is the functionality you're delivering. In this case, I'd say it is 'metadata'